pandas/tests/indexing/test_categorical.py

```diff
@@ -650,6 +650,32 @@ def test_loc_slice(self):
         expected = self.df.iloc[[2, 3, 4]]
         tm.assert_frame_equal(result, expected)
 
+    def test_reindexing_with_missing_values(self):
```

Comment:
move to pandas/tests/frame/test_alter_axes.py

you can make a new class there if you'd like and put this there

Reply:
Done